### PR TITLE
Fix errors in GH action coverage test

### PR DESCRIPTION
**Pull request by Xpirix**
- This PR is addressing the issue #322 

Summary of this PR:

- Update some packages (`django-taggit`, 'Whoosh', 'picke5') in requirements to fix errors when running tests or commands in a development environment
- Update docker-compose to fix the make error in a development environment
- Fix test_plugin_list and test_plugin_version_feedback to reflect the code. In test_plugin_version_feedback, `<div class="span8 new-feedback">` becomes `<div class="new-feedback">` because `<div class="span8 new-feedback">` doesn't exist and the test will always fail.

### CODE REVIEW

**Comment by Akiyamka** [comment]
I ran this code after the fixes and it works fine now. Tested scripts:
```
$ make build
$ make web
$ make devweb
$ make migrate
$ make devweb-runserver
$ pip install -r REQUIREMENTS-dev.txt
$ pre-commit install --config .pre-commit-config.yaml
```
`http://0.0.0.0:62202/admin/login/?next=/admin/` shows login screen.

However, when I open root `http://0.0.0.0:62202/`, I see error page
```
DoesNotExist at /
Menu matching query does not exist.
Error during template rendering
{% get_namedmenu Navigation as menu %}
```
 is this expected behaviour?

**Comment by Xpirix** [comment]
I think it's normal when DB is empty. Running `make dbseed` will probably fix this.

File: REQUIREMENTS.txt [hunk 1]
```diff
@@ -9,7 +9,12 @@ Markdown==2.3.1
 #PIL==1.1.7
 Pillow
 Pygments==2.7.4
-Whoosh==2.5.6
+
+# Updates for Django 2 & Python 3.7
+git+https://github.com/Xpirix/whoosh.git
```
Comment (dqunbp) [inline REQUIREMENTS.txt:14]: Wouldn't it be better to specify a specific commit?

File: dockerize/docker/Dockerfile [hunk 2]
```diff
@@ -12,7 +12,7 @@ RUN echo "deb http://archive.debian.org/debian stretch main contrib non-free" >
 RUN apt-get update && apt-get install -y libsasl2-dev python-dev libldap2-dev libssl-dev
 ADD REQUIREMENTS.txt /REQUIREMENTS.txt
 RUN pip install -r /REQUIREMENTS.txt
-RUN pip install uwsgi
+RUN pip install uwsgi freezegun
```
Comment (dqunbp) [inline dockerize/docker/Dockerfile:15]: Wouldn't it be better to specify a specific version?

File: dockerize/docker/REQUIREMENTS.txt [hunk 3]
```diff
@@ -21,8 +21,11 @@ django-sortable-listview
 sorl-thumbnail
 django-extensions
 django-debug-toolbar==1.11.1
-whoosh
-django-haystack
+
+# Updates for Django 2  & Python 3.7
+git+https://github.com/Xpirix/whoosh.git
```
Comment (dqunbp) [inline dockerize/docker/REQUIREMENTS.txt:26]: Is it better to point to particular commit?

File: qgis-app/REQUIREMENTS_plugins.txt [hunk 4]
```diff
@@ -21,5 +21,8 @@ django-sortable-listview
 sorl-thumbnail
 django-extensions
 django-debug-toolbar==1.11.1
-whoosh
-django-haystack
+
+# Updates for Django 2 & Python 3.7
+git+https://github.com/Xpirix/whoosh.git
```
Comment (dqunbp) [inline qgis-app/REQUIREMENTS_plugins.txt:26]: Same here

**Comment by Xpirix** [comment]
@dqunbp . Thanks for the feedback. Yes, I also think it's better to specify the commit or the version to prevent errors in the future. I've just made these fix.

**Review by dimasciput — APPROVED** [review]

**Pull request merged**